fetch/api/headers/header-setcookie.any.js

```diff
@@ -0,0 +1,223 @@
+// META: title=Headers set-cookie special cases
+// META: global=window,worker
+
+const headerList = [
+  ["set-cookie", "foo=bar"],
```

Comment:
We could have tests where set-cookie headers have empty values. 
From the draft spec, I understand that empty values are allowed?

Reply:
There is such a test further down.